README.md

```diff
@@ -73,13 +73,49 @@ Don't forget to explore our sibling project, [OllamaHub](https://ollamahub.com/)
 
 ### Installing Both Ollama and Ollama Web UI Using Docker Compose
 
-If you don't have Ollama installed yet, you can use the provided Docker Compose file for a hassle-free installation. Simply run the following command:
+If you don't have Ollama installed yet, you can use the provided bash script for a hassle-free installation. Simply run the following command:
```

Comment:
It's great to offer bash script for ease of installation, but I believe we should offer both docker compose, and bash script options in the readme.md, as there are many mac/windows users. Let's leave the compose installation section and append the bash script section right below it.

Also, I believe the bash script only works in a linux environment, please make sure to disclose that information as well. Thanks!

Reply:
Great, i'll fix that ASAP.
Also i have plans to create a .ps1 script for Windows which uses Docker Desktop over WSL2.
Next PR will support that.

Reply:
Reviewed the README.md, i've added both guides for the script and docker compose with override, default  gpu driver is nvidia and default gpu count is 1.
User can also change those values using environment variables, as specified in docker-compose.gpu.yaml file.